projects/igniteui-angular/src/lib/grids/grid/grid-toolbar.spec.ts

```diff
@@ -227,105 +238,105 @@ describe('IgxGrid - Grid Toolbar #grid', () => {
         expect(getExportOptions(fixture).length).toBe(2);
 
         getExportButton(fixture).nativeElement.click();
```

Comment:
You can change it to getExportButton(fixture).triggerEventHandler('click', new Event('click'));